README.md

```diff
@@ -1,17 +1,42 @@
-# Marker
+<h1 align="center">Marker</h1>
+<p align="center">
+    <br/>
+    <img height=200 src="misc/logo.png" alt="Marker logo"/>
+</p>
 
-A project trying to implement a stable linting API for Rust.
+<br/>
 
-## Contributing
+[![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/46214)
+![CI Status](https://github.com/rust-marker/marker/actions/workflows/rust_bors.yml/badge.svg)
+
+**A project trying to implement a stable linting API for Rust.**
```

Comment:
I just saw that this is different than the project description on GitHub.

> An experimental linting interface for Rust. Let's make custom lints a reality!

Which one do you like better?